387.First Unique Character in a String

```diff
@@ -0,0 +1,30 @@
+class Solution {
```

Comment:
add the address of the question, for example:
// https://leetcode.com/problems/first-unique-character-in-a-string/